RELEASES.md

```diff
@@ -0,0 +1,414 @@
+# Version 2.0 Syntax (2019-11-11)
+
+The ink! version 2.0 syntax has one major philosophy:
+
+> Just. Be. Rust.
+
+To accomplish this, we take advantage of all the standard Rust types and structures and use
+[attribute macros](https://doc.rust-lang.org/reference/procedural-macros.html#attribute-macros) to
+tag these standard structures to be different parts of the ink! language.
+
+Anything that is not tagged with an `#[ink(...)]` attribute tag is just standard Rust, and can be
+used in and out of your contract just like standard Rust could be used!
+
+**Every valid ink! contract is required to have at least one `#[ink(constructor)]`, at least one
+`#[ink(message)]` and exactly one `#[ink(storage)]` attribute.**
+
+Follow the instructions below to understand how to migrate your ink! 1.0 contracts to this new ink!
+2.0 syntax.
+
+## Declaring a Contract
+
+The fundamental change with the new ink! syntax is how we declare a new contract.
+
+We used to wrap the whole ink! contract into a `contract!` macro. At that point, all syntax within
+the macro could be custom, and mistakenly we used that to define custom syntax that made our code
```

Comment:
Instead of "mistakenly" I would say something like "in our first iteration". I mean, we tried that approach intentionally and it turned out that there are a number of issues with it.

Reply:
Agreed and updated.